src/types/types.ts

```diff
@@ -85,6 +85,7 @@ export interface AdjusterCellInfo {
   canBeResized: boolean;
   x?: number;
   colIdx?: number;
+  expressionColor?: ExpressionColor;
```

Comment:
Should you not add this to the actual AdjusterCellInfo instances as well?

Reply:
Not needed, in the dimension value case, we use `Cell`, so it is already included. I only specify which types the function needs.